plugin/core/edit.py

```diff
@@ -50,3 +55,17 @@ def sort_by_application_order(changes: Iterable[TextEdit]) -> List[TextEdit]:
     # we use the index in the array as the key.
 
     return list(sorted(changes, key=operator.itemgetter(0)))
+
+
+def apply_workspace_edit(window: sublime.Window, changes: Dict[str, List[TextEdit]]) -> Promise:
```

Comment:
Do you recall any server that I could use to test this with?

Reply:
Never mind. Any code action.

Reply:
clangd uses workspace/applyEdit.